purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -735,7 +745,6 @@ internal class BillingWrapper(
 
     override fun onBillingServiceDisconnected() {
         log(LogIntent.WARNING, BillingStrings.BILLING_SERVICE_DISCONNECTED.format(billingClient?.toString()))
-        retryBillingServiceConnectionWithExponentialBackoff()
```

Comment:
So on a disconnection we won't retry, but we will retry to connect on the next billing client operation correct?

Reply:
Correct

Reply:
Without back off tho, as it was before